File: executable/matlab/YIOrRd-heatmap.m
signin('test-runner', '9h29fe3l0x')

size = 50;
z = zeros(size, size);
for r = 1:size
    for c = 1:size
        z(r,c) = sqrt(r*c/size^2)
    end
end

data = {...
  struct(...
    'z', z, ...
    'scl', 'YIOrRd', ...
    'type', 'heatmap')...
};
layout = struct('title', 'YIOrRd');

response = plotly(data, struct('layout', layout, 'filename', 'YIOrRd-heatmap', 'fileopt', 'overwrite'));
plot_url = response.url

File: auto-docs/executables/python/YIOrRd_heatmap.py
import plotly.plotly as py
from plotly.graph_objs import *

py.sign_in('TestBot', 'r1neazxo9w')

import numpy as np

size = 50
z = np.zeros((size, size)).tolist()
for r in range(1, size + 1):
    for c in range(1, size + 1):
        z[r-1][c-1] = r+c

data = Data([
    Heatmap(
        z=z,
        scl='YIOrRd'
    )
])
layout = Layout(
    title='YIOrRd'
)
fig = Figure(data=data, layout=layout)

plot_url = py.plot(fig, filename='YIOrRd-heatmap', auto_open=False)